include/odp/api/spec/classification.h

```diff
@@ -523,6 +523,23 @@ typedef struct odp_cls_stats_capability_t {
 
 } odp_cls_stats_capability_t;
 
+/** Event aggregator related capabilities */
+typedef struct odp_cls_aggr_capability_t {
+	/** Supported enqueuing profile types */
+	struct {
+		/** IPv4 fragment profile */
+		uint64_t ipv4_frag : 1;
```

Comment:
It would help the reader to find additional documentation about the feature, if ODP_AEP_TYPE_IPV4_FRAG is mentioned here (especially when the enum is in another file). However, Doxygen tag "@see" should not be used as it moves tagged text to another part of the page.

Similar flag/enum documentation for IPv6 and custom flags.

Reply:
Added in v2